target/classes/META-INF/resources/components/Api/Api.js

```diff
@@ -0,0 +1,6 @@
+import axios from 'axios';
+//const baseUrl = x === 'development' ? 'http://localhost:4000' : process.env.REACT_APP_API_URL;
```

Comment:
can this be removed? let's try to only add production-necessary code and comments that are there for clarity purposes to master

Reply:
When Mufaro and I first merged our branches there was conflict with the folder structure and it took us a lot of trial and error with git to resolve the issue. One of the errors was committing every file from my local repo by accident.